host/sgx/calls.c

```diff
@@ -775,7 +775,7 @@ oe_result_t oe_ecall(
     OE_CHECK(_do_eenter(
         enclave,
         tcs,
-        OE_AEP,
+        (void (*)())OE_AEP_ADDRESS,
```

Comment:
Maybe we should change the signature of `_do_eenter` instead of type-casting here?

Reply:
Great suggestion. I have changed the signature to take in uint64_t. This reflects AEP accurately since it is not a void (void) function.